host/class/uac/usb_host_uac/include/usb/uac.h

```diff
@@ -0,0 +1,480 @@
+    UAC_FU_CONTROL_POS_AUTOMATIC_GAIN                 = 0x0040,
+    UAC_FU_CONTROL_POS_DELAY                          = 0x0080,
+    UAC_FU_CONTROL_POS_BASS_BOOST                     = 0x0100,
+    UAC_FU_CONTROL_POS_LOUDNESS                       = 0x0200,
+} uac_fu_control_pos_t;
+
+/******************************** Refer termt10.pdf ***************************************************/
+
+/**
+* @brief USB Terminal Types
+*
+* @see Table 2.1 of termt10.pdf
+*/
+typedef enum {
+    UAC_USB_TERMINAL_TYPE_USB_UNDEFINED               = 0x0100,
+    UAC_USB_TERMINAL_TYPE_USB_STREAMING               = 0x0101,
+    UAC_USB_TERMINAL_TYPE_VENDOR_SPECIFIC             = 0x01FF
+} uac_usb_terminal_type_t;
+
+/**
+* @brief Input Terminal Types
+*
+* @see Table 2.2 of termt10.pdf
+*/
+typedef enum {
+    UAC_INPUT_TERMINAL_UNDEFINED                      = 0x0200,
+    UAC_INPUT_TERMINAL_MICROPHONE                     = 0x0201,
+    UAC_INPUT_TERMINAL_DESKTOP_MICROPHONE             = 0x0202,
+    UAC_INPUT_TERMINAL_PERSONAL_MICROPHONE            = 0x0203,
+    UAC_INPUT_TERMINAL_OMNI_DIRECTIONAL_MICROPHONE    = 0x0204,
+    UAC_INPUT_TERMINAL_MICROPHONE_ARRAY               = 0x0205,
+    UAC_INPUT_TERMINAL_PROCESSING_MICROPHONE_ARRAY    = 0x0206
+} uac_input_terminal_type_t;
+
+/**
+ * @brief Output Terminal Types
+ *
+ * @see Table 2.3 of termt10.pdf
+ */
+typedef enum {
+    UAC_OUTPUT_TERMINAL_UNDEFINED                     = 0x0300,
+    UAC_OUTPUT_TERMINAL_SPEAKER                       = 0x0301,
+    UAC_OUTPUT_TERMINAL_HEADPHONES                    = 0x0302,
+    UAC_OUTPUT_TERMINAL_HEAD_MOUNTED_DISPLAY_AUDIO    = 0x0303,
+    UAC_OUTPUT_TERMINAL_DESKTOP_SPEAKER               = 0x0304,
+    UAC_OUTPUT_TERMINAL_ROOM_SPEAKER                  = 0x0305,
+    UAC_OUTPUT_TERMINAL_COMMUNICATION_SPEAKER         = 0x0306,
+    UAC_OUTPUT_TERMINAL_LOW_FREQUENCY_EFFECTS_SPEAKER = 0x0307
+} uac_output_terminal_type_t;
+
+/******************************** Refer frmts10.pdf ***************************************************/
+
+/**
+ * @brief Audio Data Format Type I Codes
+ *
+ * @see Table A 1.1 of frmts10.pdf
+ */
+typedef enum {
+    UAC_TYPE_I_UNDEFINED                              = 0x0000,
+    UAC_TYPE_I_PCM                                    = 0x0001,
+    UAC_TYPE_I_PCM8                                   = 0x0002,
+    UAC_TYPE_I_IEEE_FLOAT                             = 0x0003,
+    UAC_TYPE_I_ALAW                                   = 0x0004,
+    UAC_TYPE_I_MULAW                                  = 0x0005
+} uac_type_i_format_t;
+
+/**
+ * @brief Format Type Codes
+ *
+ */
+typedef enum {
+    UAC_FORMAT_TYPE_UNDEFINED                         = 0x0000,
+    UAC_FORMAT_TYPE_I                                 = 0x0001,
+    UAC_FORMAT_TYPE_II                                = 0x0002,
+    UAC_FORMAT_TYPE_III                               = 0x0003
+} uac_format_type_t;
+
+/**
+ * @brief Audio Class-Specific Interface Descriptor
+ *
```

Comment:
Does this `typedef` have also `see` option referring to some specific table in the docs?

Reply:
No refer table, It's the common part of Class-Specific Descriptor, used for iterate over different types of descriptors

Reply:
let's change the brief to `Audio Class-Specific Interface Descriptor Common Header`